src/Void.UI/MainForm.cs

```diff
@@ -40,7 +40,7 @@ public Message HandleViewModelEvent(VMEvent eventMsg)
         {
             if (eventMsg.IsViewPortionRendered)
             {
-                _drawings = ((VMEvent.ViewPortionRendered)eventMsg).Item2;
+                _drawings.AddRange(((VMEvent.ViewPortionRendered)eventMsg).Item2);

```

Comment:
Exactly. Because I didn't name the elements, effectively it's a tuple, and that's how you access the second element from C#. Not exactly the smoothest interop, but oh well.